src/main/charts/confluence/templates/_helpers.tpl

```diff
@@ -47,6 +47,50 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
```

Comment:
Here I've added 3 new template fragments for rendering the names of th new resources, with sensible defaults.